src/_components/Footer.jsx

```diff
@@ -84,24 +87,26 @@ export default function Footer() {
                 </div>
               </div>
             </div>
-            <div className="grid grid-cols-2 gap-20 sm:gap-60 sm:grid-cols-3 ">
+            <div className="grid grid-cols-2 gap-20 sm:grid-cols-4">
```

Comment:
I would recommend using a flexbox instead of a grid, just to be consistent with the other components and also they're better for responsiveness